public_html/index.html

```diff
@@ -1,151 +1,146 @@
+                            the legal requirements and risks of cultivating, manufacturing, delivering, transporting,
+                            storing, selling, and testing.
+                            For business owners looking to expand, she can help weigh the benefits and risks of
+                            operating in multiple states.</p>
+                    </div>
+                    <!-- End of service card -->
+                    <!-- service card -->
+                    <div class="cbailylaw__service--card">
+                        <h3>Medical-Use Cannabis</h3>
+                        <p>If you are a healthcare professional, C Baily Law LLC can answer your questions about the
+                            implications of certifying
+                            adult, pediatric, or out-of-state patients to use medical-use marijuana for your license and
+                            practice. Christine can
+                            help you understand the current state of the law, and how changes to the law may impact your
+                            future business.</p>
+                    </div>
+                    <!-- End of service card -->
+                    <!-- service card -->
+                    <div class="cbailylaw__service--card">
+                        <h3>Administrative Law</h3>
+                        <p>If you are licensed or registered with a state agency responding to an investigation, audit,
+                            or other inquiries from
+                            regulators, including if your license has been suspended or revoked, C Baily Law LLC can
+                            respond to and challenge
+                            governmental actions impacting your business, including representing you before an
+                            administrative agency or reviewing
+                            court.</p>
+                    </div>
+                    <!-- End of service card -->
+                    <!-- service card -->
+                    <div class="cbailylaw__service--card">
+                        <h3>Appellate Litigation</h3>
+                        <p>With years of experience appearing before and working for appellate courts, including two
+                            state supreme courts,
+                            Christine can advise clients on whether and when it makes sense to pursue an appeal. While
+                            she has a broad range of
+                            expertise in this area, she is uniquely qualified to represent business owners and
+                            professionals challenging adverse
+                            governmental actions on appeal.</p>
+                    </div>
+                    <!-- End of service card -->
+
+                </div>
+                <!-- End of service wrapper -->
+            </div>
+        </section>
+        <!-- End of service Section -->
+
```

Comment:
I noticed that you deleted the text from the unused sections. Another option is to put them in a temporary `div` and set the visibility to hidden, so you still have the text here in the document.